makefile

```diff
@@ -162,13 +162,8 @@ prodDistDir = $(call recipeDir,$1)/prod-dist
 
 # Tests whether recipe can be run on web browser backend
 recipes/%/web:
-> @if [ ! -d $(call webDir,$*) ]
```

Comment:
Since this rule will only be called if the target needs to be created, the directory check is unnecessary.

Unrelated to this issue, but figured to include it while I was here.

Reply:
Yeah, might as well.